packages/create-plugin/README.md

```diff
@@ -116,4 +116,4 @@ For more information see our [documentation](https://grafana.com/developers/plug
 
 ## Contributing
 
-We are always grateful for contributions! See [CONTRIBUTING.md](../CONTRIBUTING.md) for more information.
+We are always grateful for contributions! See [CONTRIBUTING.md](../../CONTRIBUTING.md) for more information.
```

Comment:
Yep there is a similar issue in the [sign-plugin readme](https://www.npmjs.com/package/@grafana/sign-plugin). Any chance you could update that one too please?

Reply:
done